docs/tutorials/task.cis-to-component-definition/transformation.md

```diff
@@ -199,11 +313,11 @@ VALID: Model /home/<user>/trestle.workspace/component-definitions/cis/component-
 ```json
 {
   "component-definition": {
```

Comment:
Would it be better if we just put a sample output rather than the full CD in the tutorial. It makes it the tutorial very big.

Reply:
Not sure what you mean by `big`? The sample output is only 2700 lines (not millions or tens of thousands). And it is only seen by clicking to expand, so the default view of the tutorial page is small in size.

But if you feel strongly, I can adjust the selected/enabled rules downward...

Reply:
I updated the selected rules to 3, which reduces the produced component-definition.json from 2700+ lines to 212.